src/internal/components/expand-toggle-button/styles.scss

```diff
@@ -40,8 +40,16 @@
   outline: 0;
   color: awsui.$color-text-interactive-default;
 
-  @include focus-visible.when-visible {
-    @include styles.focus-highlight(awsui.$space-button-focus-outline-gutter);
+  // focus ring for tree-view is different and handled in tree-item/styles
+  &:not([data-awsui-tree-view-toggle-button]) {
+    @include focus-visible.when-visible {
+      @include styles.focus-highlight(awsui.$space-button-focus-outline-gutter);
+    }
+  }
+
+  &.invisible {
+    color: transparent;
```

Comment:
Why do we set `color: transparent`? This should only affect text content of the invisible buttons, but there is no content.